archive-template.php

```diff
@@ -198,7 +198,7 @@ class="second-bar show-for-small-only center list-actions-bar"><!--  /* MOBILE V
                                                            data-field="<?php echo esc_html( $field )?>"
                                                            data-type="<?php echo esc_html( $field_options[$field]["type"] ) ?>"
                                                            name="<?php echo esc_html( $field ) ?>[query]"
-                                                           placeholder="<?php esc_html_e( "Type to Search", 'disciple_tools' ) ?>"
+                                                           placeholder="<?php echo esc_html_x( 'Type to search', 'input field placeholder', 'disciple_tools' ) ?>"
```

Comment:
There were already 3 other places where this context was for this phrase. This was the only one that didn't have it. If we don't add the context it will need to be translated twice. It won't trigger a change because it is has already been translated.

Reply:
ok.